src/ImageSharp/Formats/Jpg/Components/Block8x8F.Generated.cs

```diff
@@ -0,0 +1,55 @@
+﻿
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+
+namespace ImageSharp.Formats
+{
+	internal partial struct Block8x8F
```

Comment:
I think the purpose of these functions is simple. (e.g. transposing the block) I will trying to add some comments here.
A few words about the implementation: What I'm doing is just to unroll some loops that work on a constant-sized (8x8) block. In C# it's much more faster to access fields of a struct, than access an array with indexers. (See: Matrix classes in XNA or WPF)
First I started to implement these methods by hand, but quicky realized that using T4 I will finish my work faster, with less bugs. It's also easier to maintain & understand the T4 metaprogram, than the bloated output code.